applications/GeoMechanicsApplication/tests/1D-Consolidation_all_stages/ProjectParameters_stage2.json

```
@@ -133,23 +133,19 @@
         "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
         "process_name":  "ApplyScalarConstraintTableProcess",
         "Parameters":    {
-            "model_part_name":      "PorousDomain.Fluid_Pressure-auto-1",
+            "model_part_name":      "PorousDomain.Fluid_pressure",
             "variable_name":        "WATER_PRESSURE",
             "is_fixed":             true,
             "value":                0.0,
-            "table":                0,
-            "hydrostatic":          false,
-            "gravity_direction":    1,
-            "reference_coordinate": 0.0,
-            "specific_weight":      10000.0
+            "table":                0
         }
     }],
     "loads_process_list": [{
         "python_module": "apply_vector_constraint_table_process",
         "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
         "process_name":  "ApplyVectorConstraintTableProcess",
         "Parameters":    {
-            "model_part_name": "PorousDomain.Line_Load-auto-1",
+            "model_part_name": "PorousDomain.Line_load",
             "variable_name":   "LINE_LOAD",
             "active":          [false,true,false],
             "value":           [0.0,-1,0.0],
```

Just to be consistent with the other ProjectParameters files :)
```suggestion
            "value":           [0.0,-1.0,0.0],
```

---

Indeed, done.